NEWS

```diff
@@ -1,3 +1,10 @@
+=============================================
+NetworkManager-1.?? (not released yet)
+Overview of changes since NetworkManager-1.12
+=============================================
```

Comment:
the branch still contains some "Since: 1.12" and `NM_AVAILABLE_IN_1_12` tags that need updating.

Reply:
Fixed.